tests/topotests/bgp_evpn_rt5_leak/test_bgp_evpn_leak.py

```diff
@@ -0,0 +1,910 @@
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    evpn_check_contexts(tgen.gears["r1"], ipv4_only=True)
+    evpn_check_nexthop(tgen.gears["r2"])
+
+
+def test_evpn_ping_again():
+    """
+    Check ping between R1 and R2 is ok
+    """
+    tgen = get_topogen()
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    evpn_ping_router(tgen.gears["r1"], ipv4_only=True)
+
+
+def test_evpn_other_address_family():
+    """
+    Check the removal of an EVPN route is correctly handled
+    """
+    tgen = get_topogen()
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    config_add_ipv6 = {
+        "r2": {
+            "raw_config": [
+                "router bgp 65000 vrf vrf-101",
+                "address-family ipv6 unicast",
+                "network fd01::12/128",
+                "network fd01::2/128",
+            ]
+        }
+    }
+
+    logger.info("==== Add IPv6 again network on R2")
+    result = apply_raw_config(tgen, config_add_ipv6)
+    assert result is True, "Failed to add IPv6 network on R2, Error: {} ".format(result)
+    evpn_check_routes(tgen, "r1", "ipv6", "vrf-101", ["fd01::2/128"], expected=True)
+
+    config_no_ipv4 = {
+        "r2": {
+            "raw_config": [
+                "router bgp 65000 vrf vrf-101",
+                "address-family ipv4 unicast",
+                "no network 10.0.101.2/32",
+                "no network 10.0.101.12/32",
+            ]
+        }
+    }
+
+    logger.info("==== Remove IPv4 network on R2")
+    result = apply_raw_config(tgen, config_no_ipv4)
+    assert result is True, "Failed to remove IPv4 network on R2, Error: {} ".format(
+        result
+    )
+
+    evpn_check_routes(tgen, "r1", "ipv4", "vrf-101", ["10.0.101.2/32"], expected=False)
+    evpn_print_nexthop_rmac(tgen, "r1")
+
+
+def test_router_check_evpn_contexts_again_other_address_family():
+    """
+    Check EVPN nexthops and RMAC number  are correctly configured
+    """
+    tgen = get_topogen()
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    evpn_check_contexts(tgen.gears["r1"], ipv6_only=True)
+
+
+def test_evpn_ping_again_other_address_family():
+    """
+    Check ping between R1 and R2 is ok
+    """
+    tgen = get_topogen()
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    evpn_ping_router(tgen.gears["r1"], ipv6_only=True)
+
+
+def test_evpn_enable_l3vpn_importation():
+    """
+    Reconfigure IPv6 networks
+    Unshutdown BGP L3VPN neighbors
+    Ensure R1 receives 10 EVPN prefixes
+    """
+    tgen = get_topogen()
+    if tgen.routers_have_failure():
+        pytest.skip(tgen.errors)
+
+    tgen.gears["r2"].vtysh_cmd(
+        """
+        configure terminal
+        router bgp 65000
+        no neighbor 192.168.105.61 shutdown
+        exit
+        router bgp 65000 vrf vrf-141
+        no neighbor 192.168.106.61 shutdown
+        exit
+        """
+    )
```

Comment:
btw, is there a reason you prefer this style to apply_raw_config?

Reply:
same.